test/Project.toml

```
@@ -21,6 +22,7 @@ StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
+ADTypes = "0.2, 1.12"
```

```suggestion
ADTypes = "0.2.5, 1.12"
```
We need at least v0.2.5 because [SciMLBase.jl v2.67.0 requires that](https://github.com/SciML/SciMLBase.jl/blob/v2.67.0/Project.toml#L54)